.codecov.yml

```diff
@@ -0,0 +1,2 @@
+coverage:
+  range: "60...100"
```

Comment:
Minimum code coverage 60%?

Reply:
As our current coverage is 68% I did not want to lock the current pullrequests when leaving the standard on 70%